FindImportProfile/FindImportProfile.test.js

```diff
@@ -3,6 +3,7 @@
 import React from 'react';
 import { act } from '@testing-library/react';
 import { noop } from 'lodash';
+import { axe } from 'jest-axe';
```

Comment:
`stripes-testing` already has utils to run axe tests, along with initial configuration
https://github.com/folio-org/stripes-testing/blob/master/doc/axeTesting.md
Perhaps it's worth checking it out